connect/src/executor-api.ts

```diff
@@ -0,0 +1,119 @@
+import { type Chain, toChainId } from "@wormhole-foundation/sdk-base";
+import type { SignedQuote, TxHash } from "@wormhole-foundation/sdk-definitions";
+import axios from "axios";
+
+export enum RelayStatus {
```

Comment:
can these enums and types go in definitions instead of connect, please? 🙏

Reply:
Moved these to the definitions SDK. Decided to keep the API fetch functions in the connect SDK for now. We could potentially move those to the definitions SDK and use `fetch` instead of `axios`, although I'm not sure if fetch functions should belong in a "definitions" SDK 🤔

Reply:
Ultimately decided to move the fetch functions into the definitions SDK using fetch() instead of axios so that they can be used elsewhere from the light-weight definitions package.